test/support/test_helper.ex

```diff
@@ -68,4 +68,18 @@ defmodule TestHelper do
         do_wait_for_lambda(lambda, retries - 1, interval, counter + 1)
     end
   end
+
+  @doc """
+  test if specific tcp port is open on localhost host address.
+  """
+  def port_open?(port) do
+    case :gen_tcp.connect(to_charlist("127.0.0.1"), port, [:binary, active: false], 1000) do
```

Comment:
if you are making such a generic helper, I would like the host/ip to also be passed in, if not its too specific and could reside in the test. What do you do think? will all port tests always be done on loopback ip?

Reply:
I added the option